src/includes/performance-monitoring/configuration/vue.mdx

```diff
@@ -1,49 +1,54 @@
-
 **Vue.js**
 
 To get started with performance monitoring with Vue.js, first install these packages:
 
 ```bash
 # Using yarn
-$ yarn add @sentry/browser @sentry/integrations @sentry/apm
+$ yarn add @sentry/browser @sentry/integrations @sentry/tracing
 
 # Using npm
-$ npm install @sentry/browser @sentry/integrations @sentry/apm
+$ npm install @sentry/browser @sentry/integrations @sentry/tracing
 ```
 
+<Alert level="warning" title="Note"><markdown>
+
+We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to the tracing package](/product/performance/apm-to-tracing/)
+
+</markdown></Alert>
+
 The Vue Tracing Integration allows you to track rendering performance during an initial application load.
 
 Sentry injects a handler inside Vue's `beforeCreate` mixin, providing access to a Vue component during its life cycle stages.
-When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM Tracing integration,
-and create a new activity named `Vue Application Render`. Once the activity has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the activity as completed.
+When Sentry encounters a component named `root`, which is a top-level Vue instance (as in `new Vue({})`), we use our AM BrowserTracing integration,
+and create a new span named `Vue Application Render`. Once the span has been created, it will wait until all of its child components render, and there aren't new rendering events triggered within the configured `timeout`, before marking the span as finished.
 
 The described instrumentation functionality will give you very high-level information about the rendering performance of the Vue instance. However, the integration can also provide more fine-grained details about what actually happened during a specific activity.
 To do that, you need to specify which components to track and what hooks to listen to (you can find a list of all available hooks [here](https://vuejs.org/v2/api/#Options-Lifecycle-Hooks)). You can also turn on tracking for all the components. However, it may be rather noisy if your app consists of hundreds of components. We encourage being more specific. If you don't provide hooks, Sentry will track a component's `mount` and `update` hooks.
 
 Note that we don't use `before` and `-ed` pairs for hooks, and you should provide a simple verb instead. For example, `update` is correct. `beforeUpdate` and `updated` are incorrect.
```

Comment:
```suggestion
Note that when specifying hooks we use the simple verb rather than `before` and `-ed` pairs. For example, `destroy` is correct. `beforeDestroy` and `destroyed` are incorrect.
```

NB that I'm only pattern matching - I don't know for sure if `destroy` has a `before` and an `-ed`. I just thought that if possible it might be nice to have one that isn't the default since they already have examples of us correctly formatting the default ones.

Reply:
@kamilogorek will know.

Reply:
They all do, except `activate`. That's why we use one simple verbs instead. Here's the full mapping list - https://github.com/getsentry/sentry-javascript/blob/5ed313546dd687b637dfb20a3a28a7823278d21a/packages/integrations/src/vue.ts#L116-L123